grails-app/views/stockMovement/show.gsp

```diff
@@ -206,6 +206,14 @@
                                 ${stockMovement?.stocklist?.name?:warehouse.message(code:"default.none.label")}
                             </td>
                         </tr>
+                        <tr class="prop">
+                            <td class="name">
+                                <warehouse:message code="stockMovement.approvers.label"/>
+                            </td>
+                            <td class="value">
+                                ${stockMovement?.requisition?.approvers?.join(", ") ?: warehouse.message(code:"default.none.label")}
```

Comment:
```suggestion
                                ${stockMovement?.requisition?.approvers?.join(", ") ?: g.message(code:"default.none.label")}
```
let's use g.message, since we will want to get rid of warehouse.message